include/config/overworld.h

```diff
@@ -5,8 +5,9 @@
 #define OW_RUNNING_INDOORS          GEN_LATEST  // In Gen4+, players are allowed to run indoors.
 
 // Other settings
-#define OW_POISON_DAMAGE            GEN_LATEST // In Gen4, Pokémon no longer faint from Poison in the overworld. In Gen5+, they no longer take damage at all.
-#define OW_TIMES_OF_DAY             GEN_LATEST // Different generations have the times of day change at different times
+#define OW_POISON_DAMAGE                GEN_LATEST // In Gen4, Pokémon no longer faint from Poison in the overworld. In Gen5+, they no longer take damage at all.
+#define OW_TIMES_OF_DAY                 GEN_LATEST // Different generations have the times of day change at different times.
+#define OW_DOUBLE_APPROACH_WITH_ONE_MON FALSE      // If enabled, you can be spotted by two trainers at the same time even if you only have one eligible Pokémon in your party.
```

Comment:
Is there no precedent to enabling it by default?

Reply:
> From [Pokémon Emerald](https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9mon_Emerald_Version) to [Generation V](https://bulbapedia.bulbagarden.net/wiki/Generation_V) (as well as in [Pokémon Brilliant Diamond and Shining Pearl](https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9mon_Brilliant_Diamond_and_Shining_Pearl)), two independent Trainers who see the player at the same time will engage the player in a Double Battle. If the player has only one conscious Pokémon, the two Trainers who see the player at the same time will take turns battling the player in two Single Battles, with the second Trainer approaching and battling the player immediately after the first Trainer is defeated.

I believe not, as weird as it is

Reply:
Unfortunate.